### PR TITLE
Squash doxygen commits

### PR DESCRIPTION
**Pull request by mkstoyanov**
* using squash of commits, only keep the last doxygen page

### CODE REVIEW

File: .jenkins [hunk 1]
```diff
@@ -134,7 +134,9 @@ cmake \
                       git config user.email "noreply@ornl.gov" &&
                       git config user.name "Jenkins CI" &&
                       git commit -m "Automated documentation update" &&
-                      git push https://$GIT_CREDS@github.com/ORNL/TASMANIAN.git gh-pages'''
+                      git reset --soft HEAD^^ &&
```
Comment (dalg24) [inline .jenkins]: Why 2x ^?

Comment (mkstoyanov) [reply .jenkins]: If I understand the logic:
* move head to the last commit that should not be changed
* take everything after that commit and merge it into a single commit
Using only one ^ will redo the last commit, i.e., no change at all.

Comment (mkstoyanov) [reply .jenkins]: I've tested it.

**Comment by mkstoyanov** [comment]
Addresses a point in #417

**Comment by dalg24** [comment]
I am still confused with the two `git commit`

**Comment by mkstoyanov** [comment]
The first is a commit, the second (when combined with the soft-reset) is a `rebase -i`

I'll see if I can do that with only one commit.

**Comment by mkstoyanov** [comment]
I got it much simpler and I don't think it gets any simpler.
* when checking out, checkout all but the last commit
* commit the new page
* force-push which will overwrite the previous last commit

**Pull request merged**